lib/fast-path.js

```diff
@@ -14,17 +14,7 @@ module.exports = FastPath;
 
 // Static convenience function for coercing a value to a FastPath.
 FastPath.from = function (obj) {
-  return obj instanceof FastPath
-    // Return a defensive copy of any existing FastPath instances.
-    ? obj.copy()
-    // Otherwise use obj as the value of the new FastPath instance.
-    : new FastPath(obj);
-};
-
-FPp.copy = function () {
-  const cp = Object.create(FastPath.prototype);
-  cp.stack = this.stack.slice(0);
-  return cp;
+  return new FastPath(obj);
```

Comment:
Since `FastPath.from` is under our control we never call it with a `FastPath` instance so it can be simplified and `FPp.copy` can be removed. As with all this if we have a need for it later on it can always be added.